packages/fungible-token/bridge-fungible-token/tests/utils/setup.rs

```diff
@@ -290,19 +280,26 @@ pub(crate) async fn relay_message_to_contract(
     wallet: &WalletUnlocked,
     message: Input,
     contracts: Vec<Input>,
-    gas_coins: &[Input],
 ) -> TxId {
     let provider = wallet.provider().expect("Wallet has no provider");
     let network_info = provider.network_info().await.unwrap();
     let gas_price = network_info.min_gas_price;
-    let params: TxParameters = TxParameters::new(Some(gas_price), Some(30_000), 0);
+    let tx_policies = TxPolicies::new(Some(gas_price), None, Some(0), None, Some(30_000));
+
+    let fetched_gas_coins: Vec<Input> = provider
+        .get_coins(wallet.address(), Default::default())
+        .await
+        .unwrap()
+        .iter()
+        .map(|el| Input::resource_signed(fuels::types::coin_type::CoinType::Coin(el.clone())))
```

Comment:
el?

What is that? Let's use `lol` instead? :)

Reply:
Fixed